### ISSUE TITLE
FR: Volvo Trucks Pause Button for Video Module Block

### ISSUE THREAD

**Issue by DanielaPedrochevd** [Netcentric/vg-volvotrucks-us-rd#182]
**Agile Requirement**
As a business owner, I want to add a Pause/Play button to the Video Block so that we comply with accessibility standards. Specifically, we’re mitigating [Failure 93 of Success Criterion 1.4.2](https://www.w3.org/TR/WCAG20-TECHS/F93.html).

 **Description**
Add a Pause button for videos as in [the design](https://www.figma.com/file/Id3nd9kbfh8TpS1SfWhrW2/Volvo-Component-Library?type=design&node-id=410-1302&mode=dev).

Make functionality built with #156 generic and apply it to video module blocks

**Acceptance Criteria**

1. The Video Module has a pause button as in the [designs](https://www.figma.com/file/yv0VtHTvMoGPqkv0RAYcVp/PDP?type=design&node-id=1812-24243&mode=dev) for each of the devices (desktop, mobile, and tablet). 
2. The pause/play button should be accessible to users navigating via the keyboard, meaning it should be focusable and actionable using keyboard inputs just like with the [Hero Block](https://github.com/Netcentric/vg-volvotrucks-us-rd/issues/156). 
3. The focus style should be the same as for the scroll to top button.
4. Upon pressing the pause button, the video should immediately stop playing.
5. Once the video is paused, the [pause button changes to display a play button](https://www.figma.com/file/Id3nd9kbfh8TpS1SfWhrW2/Volvo-Component-Library?type=design&node-id=566-2770&mode=dev)
6. Videos that are in pages out of scope of redesign project should not be affected by these changes.

**Comment by DanielaPedrochevd** [comment]
related to #178

**Comment by DanielaPedrochevd** [comment]
This issue will also affect the Hero block as we would be making the functionality generic. Videos that are in pages out of scope of redesign project should not be affected by these changes.

**Comment by Andrei-Boiangiu** [comment]
confirmed as done
